<launch>
    <node pkg="motor_p" type="motor_toggle_server.py" name="motor_toggle_server" output="screen"/>
</launch>